pkgs/by-name/sw/swiftlint/package.nix

```diff
@@ -36,7 +44,7 @@ stdenvNoCC.mkDerivation rec {
     license = licenses.mit;
     mainProgram = "swiftlint";
     maintainers = with maintainers; [ matteopacini ];
-    platforms = platforms.darwin;
+    platforms = platforms.all;
```

Comment:
```suggestion
    platforms = platforms.darwin ++ platforms.linux;
```

`platforms.all` is too broad, and Swift may not even support those - I think it's best to restrict this to the actual platforms Swiftlint supports.

![image](https://github.com/user-attachments/assets/b8b1f624-1d45-4387-8ec9-4fb96f0aecd2)